packages/calcite-components/src/components/popover/resources.ts

```diff
@@ -1,4 +1,5 @@
 export const CSS = {
+  calciteFloatingUIAnim: "calcite-floating-ui-anim",
```

Comment:
Shouldn't this be imported from a shared floating-ui resource file? @jcfranco @driskull

Reply:
Yes, from https://github.com/Esri/calcite-design-system/blob/dev/packages/calcite-components/src/utils/floating-ui.ts#L319-L322.